onnxruntime/core/mlas/lib/eltwise.cpp

```diff
@@ -0,0 +1,88 @@
+/*++
+
+Copyright (c) Microsoft Corporation. All rights reserved.
+
+Licensed under the MIT License.
+
+Module Name:
+
+    eltwise.cpp
+
+Abstract:
+
+    This module implements routines to compute eltwise operations on two vectors.
```

Comment:
nit: no need to use abbreviation eltwise in comments, suggest to use `element-wise` here.